meta-mel-support/recipes-bsp/96boards-tools/96boards-tools_0.7.bb

```diff
@@ -26,7 +27,17 @@ do_install () {
 
     install -d ${D}${sbindir}
     install -m 0755 ${S}/resize-helper ${D}${sbindir}
+
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${WORKDIR}/resize-helper.sh ${D}${sysconfdir}/init.d/resize-helper.sh
+
+	sed -i -e "s:@bindir@:${bindir}:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:@sbindir@:${sbindir}:" ${D}${sysconfdir}/init.d/resize-helper.sh
+	sed -i -e "s:@sysconfdir@:${sysconfdir}:" ${D}${sysconfdir}/init.d/resize-helper.sh
```

Comment:
Using multiple sed commands rather than one is a bit pointless, but also harmless, so I'll approve regardless. Worth keeping in mind going forward, however.